### DISCUSSION TITLE
Should toolkit require Scala 3+

### DISCUSSION THREAD

**Discussion by bishabosha** [scala/toolkit#51] [Ideas]
A lot of new libraries coming out embracing the "lean scala" style support only Scala 3. If this style is to be the recommendation for newcomers, and we determine to select some of these libraries, should toolkit drop support for releasing for Scala 2.13?

**Comment by SethTisue** [comment]
Or a branch, maybe, and a new major version number. The cross built Scala 2-and-3 version, on a branch, would get version bumps only but otherwise remain frozen.
